server.js

```diff
@@ -1,20 +1,22 @@
 const express = require("express");
 require("dotenv").config();
 const mongoose = require("mongoose");
-const bodyParser = require("body-parser");
 const { graphqlHTTP } = require("express-graphql");
 const cron = require("cron").CronJob;
 
 const app = express();
-app.use(bodyParser.json());
 const schema = require("./schema/schema");
 const rootValue = require("./rootValue/rootValue");
 const PORT = process.env.PORT || 5000;
 const isAuth = require("./auth/auth");
 const getDateof = require("./helper/send_message_helper");
 
+app.delete("/hello", (req, res) => {
+  console.log("I am hit.");
+});
+
 app.use((req, res, next) => {
-  res.setHeader("Access-Control-Allow-Origin", "*");
+  res.setHeader("Access-Control-Allow-Origin", "http://localhost:3000");
```

Comment:
Does this mean your server is always going to run from your local machine?

Reply:
Have updated the client url.